### PR TITLE
💄 style(ui): Modify and repair UI layout

### PR DESCRIPTION
**Pull request by RubuJam**
#### 💻 变更类型 | Change Type

- [ ] ✨ feat
- [x] 🐛 fix
- [ ] ♻️ refactor
- [x] 💄 style
- [ ] 🔨 chore
- [ ] ⚡️ perf
- [ ] 📝 docs

#### 🔀 变更说明 | Description of Change

- 修复`助手名称`过长而导致的显示问题
- 修复模型选择中，由于`模型名称`过长导致的显示不完全
- 修复`模型ID`或`模型名称`过长，导致`设置`与`删除`按钮消失的问题
- 调整移动端的列表设置按钮显示样式
- 修改移动端的`自定义模型配置`页面
---
- Fix the display problem caused by the `Assistant Name` being too long.
- Fix the problem that the display of the `Model Name` was incomplete due to it being too long in `Model Selection`.
- Fix the problem where the `Set` and `Delete` buttons disappear when the `Model ID` or `Model Name` is too long.
- Adjust the display style of list setting button.
- Modify `custom model configuration` page on `mobile`

#### 📝 补充信息 | Additional Information

| Before | After |
|--- | ---|
| BF1 | AF1 |

---

| Before | After | 
| --- | --- |
| BF2 |  AF2 | 

---

| Before |  After |
| --- | --- |
| BF3 | AF3 | 

---

| Before | After |
| --- | --- |
| BF4 |AF4| 

---

| Before | After |
| --- | --- |
|BF5|AF5|

### CODE REVIEW

**Comment by arvinxx** [comment]
加一下 before 和 after 的对比？

File: src/app/(main)/chat/(workspace)/_layout/Mobile/ChatHeader/index.tsx
Comment (arvinxx) [inline src/app/(main)/chat/(workspace)/_layout/Mobile/ChatHeader/index.tsx:1]: 这个需要保持原状

Comment (RubuJam) [reply src/app/(main)/chat/(workspace)/_layout/Mobile/ChatHeader/index.tsx:1]: 我明天改一下

**Comment by RubuJam** [comment]
@arvinxx 助手的布局我改天再改，现在的以及修改完毕

I will modify the layout of the assistant another day. The current modification is complete.

**Comment by arvinxx** [comment]
@RubuJam model 的这个感觉不应该过长滚动，应该也是省略号比较好

**Comment by RubuJam** [comment]
如果没有过长的，就不会显示滚动，不然直接省略，还是不太方便，毕竟这个不像设置里的，还有其他方式查看

If it is not too long, no scrolling will be displayed. Otherwise, it will be omitted directly, which is still not very convenient. This is not like in the settings. There are other ways to view it.

> @RubuJam model 的这个感觉不应该过长滚动，应该也是省略号比较好

**Comment by arvinxx** [comment]
> 如果没有过长的，就不会显示滚动，不然直接省略，还是不太方便，毕竟这个不像设置里的，还有其他方式查看

Antd 的 Typography 是支持 ellpsis 用tooltip 展示的，可以试试看。https://ant-design.antgroup.com/components/typography-cn#components-typography-demo-ellipsis

File: src/components/ModelSelect/index.tsx [hunk 1]
```diff
@@ -142,7 +143,12 @@ export const ModelItemRender = memo<ModelItemRenderProps>(({ showInfoTag = true,
     <Flexbox align={'center'} gap={32} horizontal justify={'space-between'}>
       <Flexbox align={'center'} gap={8} horizontal>
         <ModelIcon model={model.id} size={20} />
-        {model.displayName || model.id}
+        <Typography.Paragraph
```
Comment (arvinxx) [inline src/components/ModelSelect/index.tsx]: 直接换行全展示吧，不需要这个 more

**Review by arvinxx — APPROVED** [review]
LGTM now, Thanks for your contribution~

**Pull request merged**